src/app/views/reports/student-report/student-report.component.html

```diff
@@ -159,6 +159,17 @@
                     {{ isReadMore ? 'read more >>':' << read less' }}
                   </button>
                 </div>
+                 <div style="margin: 20px 30px">
+                  <div style="margin-bottom: 10px">
```

Comment:
The passage should be before the question name.

Reply:
Done